tpgtools/overrides/eventarc/samples/channel/meta.yaml

```diff
@@ -0,0 +1,5 @@
+# meta.yaml
+#
+#
+test_hide:
```

Comment:
Will do so

Reply:
Done

Reply:
More specifically, what we want to record are the constraints that meant we had to handwrite it. I think that's because we had to use bootstrapped KMS resources?

Reply:
Done